test/conformance/tests/sriov_operator.go

```diff
@@ -1968,3 +1982,32 @@ func setSriovOperatorSpecFlag(flagName string, flagValue bool) {
 
 	}
 }
+
+func getNodeDrainState() (bool, error) {
+	sriovOperatorConfg := &sriovv1.SriovOperatorConfig{}
+	err := clients.Get(context.TODO(), runtimeclient.ObjectKey{Name: "default", Namespace: operatorNamespace}, sriovOperatorConfg)
+	return sriovOperatorConfg.Spec.DisableDrain, err
+}
+
+func setDisableNodeDrainState(state bool) error {
+	sriovOperatorConfg := &sriovv1.SriovOperatorConfig{}
+	err := clients.Get(context.TODO(), runtimeclient.ObjectKey{Name: "default", Namespace: operatorNamespace}, sriovOperatorConfg)
+	if err != nil {
+		return err
+	}
+	sriovOperatorConfg.Spec.DisableDrain = state
+	err = clients.Update(context.TODO(), sriovOperatorConfg)
+	if err != nil {
+		return err
+	}
+	err = clients.Get(context.TODO(), runtimeclient.ObjectKey{Name: "default", Namespace: operatorNamespace}, sriovOperatorConfg)
+	return err
+}
+
+// Clean restores modified env resources to original state
+func Clean() error {
```

Comment:
if you move the functions above to the cluster.go and this one to the clean.go it will be better to also use it in the clean.All and cnf-tests in the future